package.json

```diff
@@ -9,7 +9,9 @@
     "main": "./lib/index.js",
     "types": "./lib/index.d.ts",
     "files": [
-        "lib"
+        "lib",
+        "src",
+        "style"
     ],
     "scripts": {
         "build": "tsc && yarn lint",
```

Comment:
Please remove the entire `./node_modules` folder in below `clean` script. Not only `./node_modules/.tmp`.

Reply:
Instead I introduced `clean:build` and `clean` to separate cleaning everything and cleaning only build information.